File: app/src/main/java/com/stupidtree/cloudliter/data/model/Yunmoji.java
package com.stupidtree.cloudliter.data.model;

public class Yunmoji {
    private int imageID;
    private String lastStr;

    public Yunmoji(int imageID) {
        this.imageID = imageID;
    }

    public int getImageID() {
        return imageID;
    }

    public String getLastname(int position) {
        if (position <= 8) {
            lastStr = "[y00" + Integer.valueOf(position+1).toString() + "]";
        } else {
            lastStr = "[y0" + Integer.valueOf(position+1).toString() + "]";
        }
        return lastStr;
    }
}

File: app/src/main/java/com/stupidtree/cloudliter/data/model/Yunmoji.kt
package com.stupidtree.cloudliter.data.model

class Yunmoji(val imageID: Int,val description:String) {
    private var lastStr: String? = null

    fun getLastName(position: Int): String {
        lastStr = if (position <= 8) {
            "[y00" + Integer.valueOf(position + 1).toString() + "]"
        } else {
            "[y0" + Integer.valueOf(position + 1).toString() + "]"
        }
        return lastStr as String
    }

}